internal/cli/users.go

```diff
@@ -1,18 +1,63 @@
 package cli
 
 import (
+	"encoding/json"
+	"errors"
 	"fmt"
 
 	"github.com/auth0/auth0-cli/internal/ansi"
+	"github.com/auth0/auth0-cli/internal/auth0"
+	"github.com/auth0/auth0-cli/internal/prompt"
 	"github.com/spf13/cobra"
 	"gopkg.in/auth0.v5/management"
 )
 
+
 var (
 	userID = Argument{
 		Name: "User ID",
 		Help: "Id of the user.",
 	}
+	userConnection = Flag{
+		Name:       	"Connection",
+		LongForm:   	"connection",
+		ShortForm:  	"c",
+		Help:       	"Name of the connection this user should be created in.",
+		IsRequired: 	true,
+		AlwaysPrompt:	false,
+	}
+	userEmail = Flag{
+		Name:       	"Email",
+		LongForm:   	"email",
+		ShortForm:  	"e",
+		Help:       	"The user's email.",
+		IsRequired: 	true,
+		AlwaysPrompt: 	false,
```

Comment:
```suggestion
```
False is the default value.

Reply:
`AlwaysPrompt ` is set to false. are you suggesting i changed them to true? it wasn't clear